### DISCUSSION TITLE
Clickjacking protection

### DISCUSSION THREAD

**Discussion by AronNovak** [community/community#32112] [Other Feature Feedback, Questions, & Ideas]
https://clickjacker.io/test?url=www.gizra.com

We have a site that's on GH Pages (https://www.gizra.com/), and recently realized that there's no protection against clickjacking.
Based on forum threads, it seems in the past, `X-Frame-Options` header was added to all content served via Pages.
Was it changed recently?
Is there any chance that we could have that protection back? (alongside with CSP Header - `Frame-Ancestors` perhaps)

Thanks!

**Comment by yoannchaudet** [comment]
We only support `X-Frame-Options` for [private pages sites](https://docs.github.com/en/enterprise-cloud@latest/pages/getting-started-with-github-pages/changing-the-visibility-of-your-github-pages-site) today.

Nothing has changed lately in that regard.

We do not support custom headers customization yet.

**Comment by janekmi** [comment]
Any plans to support either `X-Frame-Options` or `Content-Security-Policy` for the public pages sites in the foreseeable future?

**Comment by sadespresso** [comment]
Do we have an update on this? It's 2025...
